xbmc/GUIPassword.h

```diff
@@ -64,23 +65,27 @@ class CGUIPassword : public ISettingCallback
   void LockSources(bool lock);
   void RemoveSourceLocks();
   bool IsDatabasePathUnlocked(const std::string& strPath, VECSOURCES& vecSources);
-  /*! \brief Tests if the user is allowed to access the path by looking up the matching Mediasource
-   \param strPath The folder path to access
-   \param strType The type of share being accessed, e.g. "music", "video", etc. See CSettings::UpdateSources()
-   \return If access is granted, returns \e true
-   */
-  bool IsMediaPathUnlocked(const std::string& strPath, const std::string& strType);
+
+  const std::string& GetMediaSourcePath() const { return m_strMediaSourcePath; }
+  void SetMediaSourcePath(const std::string& MediaSourcePath);
 
   void OnSettingAction(std::shared_ptr<const CSetting> setting) override;
 
   bool bMasterUser;
   int iMasterLockRetriesLeft;
-  std::string strMediasourcePath;
 
 private:
+  /*! \brief Helper function to test if the user is allowed to access the path
+   by looking up the matching Mediasource. Used internally by CheckMenuLock.
+   \param profileManager instance passed by ref. see CGUIPassword::CheckMenuLock
+   \param strType The type of share being accessed, e.g. "music", "video", etc.
+   \return If access is granted, returns \e true
+   */
```

Comment:
those priv declarations don't appear in the codedocs. fine if i remove the doxy?

Reply:
No, keep it. Every documentation, given content use correct, is valuable. If there are problems with doxygen, root car should be fixed. But not in this PR.